pitch.md

```diff
@@ -1,20 +1,32 @@
-# Project Title (replace me)
+# Bridge Hand DSL
 
 ## Describe your project in five words
+Easy, powerful, bridge hand language
 
 ## Audience
 _Who are you helping, with your language? What is that person's
 experience like now? How would their experience be better if you could help 
 them?_
 
+I'm helping bridge players.  This would help bridge players who want to study
+hands, share hands with other players, and just keep a database of hands. Also
+could help bridge organizations that help teach bridge players by providing an
+easy way to share hands with others.
 
 ## Why a language?
 _Why is a **DSL** appropriate for your user(s)? How does it address the need?_
 
+Bridge already has a specialized language to talk about hands. You might here a bridge
+player say "I was red on white and held ace king fifth, four small, ace king queen
+and a stiff jack." Additionally there is a lot of information to organize and
+describe about a bridge hand so having a nice langauge to do so would be useful.
 
 ## Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I think bridge is a really exciting game, but it is a bit behind the times in
+technology.  This would be a way to make a DSL for a domain I really care about
+and if I make it well it's possible people might want to use it.
```

Comment:
It may be more persuasive, especially in terms of a short pitch or introduction to the presentation, to include some experience of how you came up with it--have you had trouble trying to teach someone bridge without this or talking with fellow bridge players?